subsys/bluetooth/shell/CMakeLists.txt

```diff
@@ -22,8 +22,7 @@ zephyr_library_sources_ifdef(
   rfcomm.c
   )
 
-if(CONFIG_BT_CTLR)
-  if(CONFIG_BT_LL_SW_LEGACY OR CONFIG_BT_LL_SW_SPLIT)
+if(CONFIG_BT_CTLR AND CONFIG_BT_LL_SW_SPLIT)
     zephyr_library_sources(
       ll.c
```

Comment:
Not sure what to do here @cvinayak, since this file is actually removed

Reply:
You have to put back this ll.c which is shell cmds, most like the test*, advx and scanx is used by me for testing on split controller.